Sourcery/Utils/Array+Parallel.swift

```diff
@@ -5,6 +5,8 @@
 
 import Foundation
 
+private let disableParallelMapForEasierDebugging = false
```

Comment:
honestly this is critical improvement as it makes debugging so much more managable